lxd/storage/backend_lxd.go

```diff
@@ -2641,3 +2644,99 @@ func (b *lxdBackend) createStorageStructure(path string) error {
 
 	return nil
 }
+
+// BackupInstanceConfig writes the instance's config to the backup.yaml file on the storage device.
+func (b *lxdBackend) BackupInstanceConfig(inst instance.Instance, op *operations.Operation) error {
```

Comment:
What do you think of `UpdateInstanceBackupFile`? Might be clearer and better follow our naming scheme.

Reply:
@stgraber yep works for me, will change and rebase.